lib/streamlit/elements/arrow_altair.py

```diff
@@ -49,6 +49,12 @@
 if TYPE_CHECKING:
     from streamlit.delta_generator import DeltaGenerator
 
+# Create and enable streamlit theme
+STREAMLIT_THEME = {"embedOptions": {"theme": "streamlit"}}
+
+# This allows to use alt.themes.enable("streamlit") to activate Streamlit theme.
+alt.themes.register("streamlit", lambda: {"usermeta": STREAMLIT_THEME})
```

Comment:
Hmm, not sure if this is very useful with streamlit theme being activated as default. I think the only use is that if `streamlit` is activated as global theme, `theme=None` will also lead to using the `streamlit` theme, or?

Btw. can you add a screenshot test here to make sure that the global theming is doing what is expected

Reply:
> Btw. can you add a screenshot test here to make sure that the global theming is doing what is expected
Yep I'll add a test. 
> Hmm, not sure if this is very useful with streamlit theme being activated as default. I think the only use is that if streamlit is activated as global theme, theme=None will also lead to using the streamlit theme, or?
yes this is just to make it so streamlit can be activated on a global theme. Arnaud specifically wanted this so he didn't have to change his apps. If you have qualms about merging this, let me know and I can bring them up with Arnaud.

Reply:
It probably does not hurt to have it in, so I think it's fine 👍

Reply:
I went to go test this and tried doing `alt.themes.enable('streamlit')` and whenever I do a st_altair_chart(...), it will automatically turn on streamlit theme because streamlit is the default.
As for when do st_altair_chart(..., theme=None), this automatically removes the streamlit theme. It feels like there is no need to add a test for this command?

Reply:
Hmm, but I think in this case, it's not really working as expected: I would expect that setting the streamlit theme globally (via `alt.themes.enable('streamlit')`) and using `theme=None` would result in the streamlit theme as well.

Reply:
The reason is that `theme=None` just means that we are not applying/overwriting any theme but instead using the globally defined theme. Which would be `streamlit` in case of `alt.themes.enable('streamlit')`.